src/resmom_win/mom_stage_file.c

```diff
@@ -84,6 +84,25 @@ size_t cred_len = 0;
 char mom_host[PBS_MAXHOSTNAME+1] = {'\0'};
 struct cphosts *pcphosts = 0;
 int cphosts_num = 0;
+static int is_file_open;
+static char id[] = "pbs_stage_file";
+
+/**
+ * @brief
+ * 	print_or_log_err - log/print error messages
+ *
+ *  @param[in]	err_msg	 - error message to log
+ *
+ * @return	void
+ */
+void
+print_or_log_err(char *err_msg)
+{
+	if (is_file_open)
+		log_err(-1, id, err_msg);
+	else
+		printf("%s:%s\n", id, err_msg);
```

Comment:
since we are printing an error. should it go to stderr?

Reply:
Earlier also we were doing like this only, anyway it's a backup only